.gitignore

```diff
@@ -8,3 +8,5 @@ venv/
 **__pycache__/
 **db.sqlite3
 media
+pillow
```

Comment:
The entry `pillow` seems to be incorrect. If you intend to ignore a specific directory or file related to the Pillow library, you might want to specify the correct path or pattern. Otherwise, this line might not have the intended effect.